control-plane-operator/hostedclusterconfigoperator/controllers/drainer/drainer.go

```diff
@@ -100,6 +100,12 @@ func (r *Reconciler) handleNodeDrainRequest(ctx context.Context, node *corev1.No
 		Ctx:    ctx,
 	}
 
+	if noderefutil.IsNodeUnreachable(node) {
```

Comment:
can we please create our own function for IsNodeUnreachable? we want to remove deps as much as possible to only the API.

Reply:
done